client/logs/bdqmlv3oo3suave89mr0.log

```diff
@@ -0,0 +1 @@
+sajt
```

Comment:
I don't think these files supposed to be under version control. Missing a `.gitignore` directive?